cli/compose/types/types.go

```diff
@@ -268,74 +401,79 @@ func (u UnitBytes) MarshalYAML() (interface{}, error) {
 	return fmt.Sprintf("%d", u), nil
 }
 
+// MarshalJSON makes UnitBytes implement json.Marshaler
+func (u UnitBytes) MarshalJSON() ([]byte, error) {
```

Comment:
Is this really needed? The default JSON marshaller does not work on this type?

Reply:
This type would be unmarshaled into a number type by default, whereas the schema enforces the string type for these fields.